source/patterns/@aws-solutions-constructs/aws-iot-s3/test/integ.iot-s3-new-encrypted-bucket.ts

```diff
@@ -21,7 +21,8 @@ import { BucketEncryption } from "@aws-cdk/aws-s3";
 const app = new App();
 const stack = new Stack(app, generateIntegStackName(__filename));
 const existingKey = new kms.Key(stack, `existingKey`, {
-  enableKeyRotation: true
+  enableKeyRotation: true,
+  removalPolicy: RemovalPolicy.DESTROY
 });
 
 const props: IotToS3Props = {
```

Comment:
This should have s3AccessLogs on, which is good - but we need to provide props for the logging bucket so that it gets cleaned up.

Reply:
updated to clean up the access logs bucket